city/models.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.orm import relationship
+
+from database import Base
+from temperature import models
+
+
+class City(Base):
+    __tablname__ = "city"
```

Comment:
There is a typo in `__tablname__`. It should be `__tablename__` to correctly define the table name for the SQLAlchemy model.